src/games/strategy/engine/lobby/server/headless/HeadlessLobbyConsole.java

```diff
@@ -29,8 +30,7 @@
   private final LobbyServer server;
   private final PrintStream out;
   private final BufferedReader in;
-  @SuppressWarnings("deprecation")
-  private final String startDate = new Date().toGMTString();
+  private final String startDate = new SimpleDateFormat().format(new Date());
```

Comment:
Sigh, this looks a bit too familiar. There is some massive copy paste when it comes to some of the higher layers of code.

Regardless, it looks like a util function somewhere to get a GMT date string would be useful, since it is used in multiple places.